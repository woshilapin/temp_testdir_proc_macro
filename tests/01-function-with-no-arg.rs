use with_tempdir_procmacro::with_tempdir;

#[with_tempdir]
fn no_arg_fn() {
    unimplemented!()
}

fn main() {
    assert!(true);
}
