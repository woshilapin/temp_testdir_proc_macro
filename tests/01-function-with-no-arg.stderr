error: Function must take one argument of type `&std::path::Path`
 --> $DIR/01-function-with-no-arg.rs:4:4
  |
4 | fn no_arg_fn() {
  |    ^^^^^^^^^
